renovate.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":dependencyDashboard",
```

Comment:
This line enables the dependency dashboard ([ref](https://docs.renovatebot.com/key-concepts/dashboard/)).

The dashboard shows an overview of the state of the repository's dependencies.